lighthouse-core/audits/unused-css-rules.js

```diff
@@ -127,7 +132,7 @@ class UnusedCSSRules extends Audit {
     const unusedRatio = (unused / usage.length) || 0;
     const results = Object.keys(indexedSheets).map(sheetId => {
       return UnusedCSSRules.mapSheetToResult(indexedSheets[sheetId]);
-    });
+    }).filter(Boolean);
```

Comment:
Yeah I guess this is a pattern I'm used to, but is there a reason we don't use `Boolean` anywhere? `!!` seems to be preferred for casting throughout.